spec/onlyoffice_logger_helper_spec.rb

```diff
@@ -17,4 +17,14 @@
     expect { described_class.log('Hello', 31) }
       .to output(/31/).to_stdout
   end
+
+  it 'Output have code green color if it specified' do
+    expect { described_class.green_log('test') }
```

Comment:
1. I think you should add two test for each method
First test check that `'test'` word is outputed
Second test that color string is present
2. YOu should use not /31m/ but use GREEN_COLOR_CODE constant